warpgate-sso/src/sso.rs

```diff
@@ -60,33 +58,13 @@ async fn make_client(
     config: &SsoInternalProviderConfig,
     http_client: &reqwest::Client,
 ) -> Result<
-    Client<
-        EmptyAdditionalClaims,
-        CoreAuthDisplay,
-        CoreGenderClaim,
-        CoreJweContentEncryptionAlgorithm,
-        CoreJsonWebKey,
-        CoreAuthPrompt,
-        StandardErrorResponse<CoreErrorResponseType>,
-        StandardTokenResponse<
-            IdTokenFields<
-                EmptyAdditionalClaims,
-                EmptyExtraTokenFields,
-                CoreGenderClaim,
-                CoreJweContentEncryptionAlgorithm,
-                CoreJwsSigningAlgorithm,
-            >,
-            CoreTokenType,
-        >,
-        StandardTokenIntrospectionResponse<EmptyExtraTokenFields, CoreTokenType>,
-        CoreRevocableToken,
-        StandardErrorResponse<RevocationErrorResponseType>,
-        EndpointSet,
-        EndpointNotSet,
-        EndpointNotSet,
-        EndpointNotSet,
-        EndpointMaybeSet,
-        EndpointMaybeSet,
+    CoreClient<
```

Comment:
Lol was way more clearer. I was a bit shame of that part of code, was generated by my text editor (Zed)...
With your update is more clear that I could use CoreClient type with its generic